core/hash/shared/to_s.rb

```diff
@@ -84,4 +84,10 @@
     expected = ruby_version_is("3.4") ? "{true => false}" : "{true=>false}"
     { true => false }.to_s.should == expected
   end
+
+  ruby_version_is "3.4" do
+    it "" do
+      { "needs-quotes": 1 }.send(@method).should == '{"needs-quotes": 1}'
+    end
+  end
```

Comment:
Missing title

Reply:
Added the title (I often start with a blank string just to try something, then later go back to add the title once I'm sure what the behaviour is, I guess I forgot step 2)